client/opts.go

```diff
@@ -27,6 +27,9 @@ type (
 		key      *ecdsa.PrivateKey
 		session  *session.Token
 		bearer   *token.BearerToken
+		// network magic is a client config, but it's convenient to copy it here (see v2MetaHeaderFromOpts)
+		// the value remains the same between calls
+		netMagic uint64
```

Comment:
could be here sometimes in future use cases when a caller may want to change the magic in `callOptions`?

Reply:
Purely theoretically, yes, but for such a need, I would expect the presence of several client instances.